### PR TITLE
Add Projects: Entitas Sync Framework

### PR DESCRIPTION
**Pull request by RomanZhu**
A very simple to use networking framework I did while working on a turn-based game

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -115,6 +115,7 @@ Please feel free to contribute after reading [contribution guidelines](CONTRIBUT
   - [AuthoritativeMovementExample](https://github.com/Relic/AuthoritativeMovementExample) - Server authoritative movement project using Forge Networking for Unity.
   - [Angry Bots Multiplayer](https://assetstore.unity.com/packages/templates/photon-angry-bots-multiplayer-showcase-1917) - Photon PUN-based multiplayer top-down shooter project.
   - [ArenaGame](https://github.com/NFMynster/ArenaGame) - Forge Networking-based multiplayer FPS game as an example project.
+  - [Entitas Sync Framework](https://github.com/RomanZhu/Entitas-Sync-Framework) - Networking framework for Entitas ECS. Targeted at slow-paced genres.
```
Comment (MFatihMAR) [inline README.md]: I had a look at the project and it actually looks promising!
However, please don't get me wrong but I don't think it's "awesome" at this stage.
I will be keeping eye on your project for a while and hoping to see it growing.
Please feel free to open a PR again in the future when you think it's mature enough.
Maybe I could even add it in the future.

Thanks again and good luck with your project! :)

**Review by 0xFA11 — COMMENTED** [review]
I had a look at the project and it actually looks promising!
However, please don't get me wrong but I don't think it's "awesome" at this stage.
I will be keeping eye on your project for a while and hoping to see it growing.
Please feel free to open a PR again in the future when you think it's mature enough.
Maybe I could even add it in the future — good luck! :)

**Comment by RomanZhu** [comment]
What do you mean by "awesome" feature?
What would you add to make it "awesome"?

**Comment by nxrighthere** [comment]
@MFatihMAR So following by your logic regarding "awesome" stuff, UE4 basic replication example under 300 SLOC is awesome, but full-featured ECS framework with auto-sync is not?

Or garbage SFX2X SDK examples are awesome? Really?

**Comment by madebynoxc** [comment]
This list contains obsolete stuff like UNET which is not maintained anymore and you are rejecting more promising, optimized and modern approaches.

**Comment by 0xFA11** [comment]
Hmm, yeah I think I really made a mistake there, apologies @RomanZhu :(
Regarding the existing standard of the quality (as @nxrighthere and @NoxCaos pointed) it's not fair.
I was at work while reviewing the PR, maybe I just judging it too much — I shouldn't be a gatekeeper.

I really messed this up, I'm happy to have it in with a good description.
Please let me know if you want to make another PR or I could simply add it.

**Comment by RomanZhu** [comment]
Sure, you can add it
 ```
 - [Entitas Sync Framework](https://github.com/RomanZhu/Entitas-Sync-Framework) - Framework for Entitas ECS. Targeted at turnbased games or other slow-paced genres.
````

**Pull request closed without merging**